tempesta_fw/http_parser.c

```diff
@@ -68,6 +68,8 @@ enum {
  */
 #define IS_CR_OR_LF(c) (c == '\r' || c == '\n')
 
+#undef isspace
+#define isspace(c) (c == ' ' || c == '\t')
```

Comment:
It is correct for current format of the HTTP-date. In one place of the date-processing it replaced by the IS_CR_OR_LF.